### DISCUSSION TITLE
Random database_prefix (source friends-of-presta)

### DISCUSSION THREAD

**Discussion by tajriba** [PrestaShop/PrestaShop#32418] [Ideas]
### The problem

Security idea: Change the default database prefix ps_ by a new longer arbitrary prefix

### The solution

Change in the install wizard a random prefix like: ps234343A_ instead of ps_

### Additional context

Nethertheless, be warned that this is useless against blackhat with DBA senior skilled because of a design vulnerability in DBMS

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
I like the idea 😊

**Reply by tajriba** [reply]
Thanks! This idea is not mine, the source of this security idea is from [Friends-Of-Presta Security Advisories].
I thought, let's put in on the right place at Github.
Have a nice day!!!
